docs/install/macos/automated.rst

```diff
@@ -2,7 +2,8 @@
 
 
 {{ apple_icon }} --- Automated Installation
---------------------------------------------
+============================================
```

Comment:
this will increase the font size in the header, no? Is this what we want? Why did you do this change?

Reply:
It did not increase the font size in mine, they're the same size. However, the Contents section on the right side bar seems to be affected by not following the (h1, h2, h3..), that's why we were seeing those blue headings there.

If you want to have the previous structure I can easily fix the .css :)

Reply:
Ok, thanks for that! Don't change it then!